verification/curator-service/ui/src/components/LinelistTable.tsx

```diff
@@ -82,13 +81,13 @@ interface LocationState {
     newCaseIds: string[];
     editedCaseIds: string[];
     bulkMessage: string;
-    searchQuery: string;
 }
 
 interface Props
     extends RouteComponentProps<never, never, LocationState>,
         WithStyles<typeof styles> {
     user: User;
+    search: string;
```

Comment:
`search?` would make you able to not specify it everywhere when instantiating the component (but perhaps involves a bunch of checks to be done here as well so might not be beneficial in the end...)

Reply:
Yea, I think that'd only really be useful for tests, where it's not too difficult to add it. Don't think it's worth the extra checks in non-test code.